package.json

```diff
@@ -13,12 +13,14 @@
     "morgan": "^1.6.1",
     "murmurhash-js": "^1.0.0",
     "promise": "^7.0.4",
-    "socket.io": "^1.3.7",
     "uuid": "^2.0.1"
   },
   "devDependencies": {
     "request": "^2.67.0",
     "request-promise": "^1.0.2",
     "yargs": "^3.31.0"
+  },
+  "scripts": {
```

Comment:
i think this is set by default, you only need to do this is you want the npm install to run